sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -325,7 +325,7 @@
       "post": {
         "summary": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language.",
         "description": "Query an IoT hub to retrieve information regarding device twins using a SQL-like language. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-query-language for more information. Pagination is supported. This returns information about device twins only.",
-        "operationId": "RegistryManager_QueryIotHub",
+        "operationId": "Devices_Query",
```

Comment:
This API should have its own subclient "Query"

Reply:
Something like "Query_Twins"

Reply:
How about Devices_QueryTwins? Do we want to create Query client just for twins?

Reply:
We will have a Query namespace in our CL that references this single API. As such, the naming here needs to be more general than "Devices_"

Since it is for querying twins (including modules), not querying devices or device twins, I stand by "Query_Twins"

Reply:
Ok that clarification helps now. So as I understand we have just one API querying both device and module twins. Should our description also include that?
I'll change it to call Query_Twins.